src/Symfony/Component/ExpressionLanguage/Node/GetAttrNode.php

```diff
@@ -34,19 +34,20 @@ public function __construct(Node $node, Node $attribute, ArrayNode $arguments, i
 
     public function compile(Compiler $compiler)
     {
+        $nullSafe = $this->nodes['attribute'] instanceof ConstantNode ? $this->nodes['attribute']->isNullSafe : false;
```

Comment:
`this->nodes['attribute'] instanceof ConstantNode && $this->nodes['attribute']->isNullSafe;`

Reply:
[Done!](https://github.com/symfony/symfony/pull/45795/commits/bffdcd22cabac5e105697b44e8e34c080dbe5f39)